### ISSUE TITLE
Make Wayland compatible

### ISSUE THREAD

**Issue by EUA** [EUA/wxHexEditor#143]
With Wayland, wxHexEditor is not so usable.

**Comment by flozz** [comment]
For those who are seeking for a quick workaround, you can make it works by forcing GDK to use its X11 backend instead of Wayland:

    GDK_BACKEND=x11 wxHexEditor

:)


[more explanations](https://blog.flozz.fr/2019/11/01/wxhexeditor-probleme-de-rafraichissement-sous-ubuntu-1910-eoan-ermin/)

**Comment by istepaniuk** [comment]
On Wayland, on 0.24, `GDK_BACKEND=x11` is necessary for the hex panel to refresh on scroll.

**Comment by EUA** [comment]
@istepaniuk yup, it's fixed that. You can report any other Wayland issues if you find any.
